subworkflows/local/prepare_genome.nf

```diff
@@ -13,6 +13,7 @@ include {
     GUNZIP as GUNZIP_ADDITIONAL_FASTA } from '../../modules/nf-core/modules/gunzip/main'
 include { UNTAR as UNTAR_BWA_INDEX
           UNTAR as UNTAR_DRAGMAP      } from '../../modules/nf-core/modules/untar/main'
+include { SAMTOOLS_FAIDX } from '../../modules/nf-core/modules/samtools/faidx/main'
```

Comment:
Obligatory indent-is-off comment 😅

Reply:
I was debating whether to fix all of those in this PR, but I think I'm going to make another one just for this.